roamer/session.py

```diff
@@ -33,5 +34,21 @@ def process(self, output):
         engine = Engine(self.directory, self.edit_directory)
         engine.compile_commands()
         print(engine.commands_to_str())
+        if engine.commands:
+            answer = 'y'
+            if not self.skipapproval:
```

Comment:
This can be simplified as follows?
```
        if engine.commands and not self.skipapproval:
```

* We can combine the nested `if` checks onto a single line.  
* `answer` does not need to be assigned 'y' here as it will always be assigned further down before it is checked.  So we can remove line 38.

Reply:
You're right. It was incremental update of code. Just added two features, one by one..
- [x]  Done